tests/commands/test_pack.py

```diff
@@ -0,0 +1,418 @@
+# Copyright 2020 Canonical Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For further info, check https://github.com/canonical/charmcraft
+
+import logging
+import pathlib
+import zipfile
+from argparse import Namespace
+from unittest.mock import patch
+
+import pytest
+import yaml
+
+from charmcraft.cmdbase import CommandError
+from charmcraft.commands import pack
+from charmcraft.commands.pack import (
+    PackCommand,
+    build_zip,
+    get_paths_to_include,
+)
+
+
+@pytest.fixture
+def bundle_yaml(tmp_path):
+    """Create an empty bundle.yaml, with the option to set values to it."""
+    bundle_path = tmp_path / 'bundle.yaml'
+    bundle_path.write_text("{}")
```

Comment:
redundant
```suggestion
    bundle_path.write_text("{}")
```

Reply:
I like the added simplicity of just calling it. However I prefer the explicitness of passing the value you want there. By default it's empty.

Changed it to do all this.

Reply:
looks great, thanks!